src/app/components/Card/Card.tsx

```diff
@@ -42,3 +44,7 @@ const Memories = styled.p`
   padding: 0.5rem 1rem;
   margin: 0;
 `;
+
+const Date = styled.p`
+  padding: 0.5rem 0.75rem;
```

Comment:
I don't think it's a good idea to use relative sizing in things like padding, margins, and such. 
The problem with that is that if a user changes his standard font size (like many people with sight disabilities do), it would automatically increase the padding/margin as well, which makes the app not very accessible. I don't think it's your intention to have the user scroll through half a page of padding before getting to the content. :)